src/main/java/tutorlink/TutorLink.java

```diff
@@ -49,18 +49,19 @@ public static void main(String[] args) {
 
     private static void setUpLogger() {
         LogManager.getLogManager().reset();
-        LOGGER.setLevel(Level.WARNING);
+        Logger rootLogger = Logger.getLogger("");
+        rootLogger.setLevel(Level.WARNING);
```

Comment:
I like how you set up an instance of Logger object rootLogger to specify clearly that this logging takes place in the root.